src/js/components/support-button/SupportButton.jsx

```diff
@@ -1,39 +1,47 @@
 import React, { useEffect } from 'react';
 
+import { faLifeRing } from '@fortawesome/free-regular-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import axios from 'axios';
 import PropTypes from 'prop-types';
 import { useChat } from 'react-live-chat-loader';
 
 import Translate from 'utils/Translate';
 
 import './SupportButton.scss';
 
-const SupportButton = ({ buttonColor, text, defaultText }) => {
+const SupportButton = ({ text, defaultText }) => {
   const [, loadChat] = useChat();
+
   useEffect(() => {
-    loadChat({ open: false });
+    loadChat({ open: false }); // instantiate `window` object
+    axios.get('/openboxes/api/helpscout/configuration/')
+      .then((response) => {
+        window.Beacon('config', response.data);
+      });
   }, []);
 
```

Comment:
Hi Matt, got interested in the topic - if you want the React to trigger immedietaly the changes of the language, you can try this, maybe it works:

`useEffect(() => {
    axios.get('/openboxes/api/helpscout/configuration/')
      .then((response) => {
        console.log(response.data);
        window.Beacon('config', response.data);
      });
  }, [locale]);`

and in the previous `useEffect` leave just **loadChat**